myosuite/envs/myo/myochallenge/run_track_v0.py

```diff
@@ -449,6 +471,14 @@ def _get_fallen_condition(self):
             else:
                 return 0
 
+    def get_limitfrc(self, joint_name):
```

Comment:
can you add documentation why this measurement is related to pain and why it is calculated in this way?

Reply:
This function calculates joint overextension force based on how much the joint limits are violated, and assumes that violating joint limits would create discomfort and pain. I can put that into the documentation